src/pixelHuntStudy.ts

```diff
@@ -64,44 +66,26 @@ async function handlePixel(details: browser.WebRequest.OnBeforeRequestDetailsTyp
 
     const hasFacebookLoginCookies = (has_c_user && has_xs);
 
-    // Attempt to associate this pixel tracker sighting with a WebScience Page ID.
-    const pageVisits = (await browser.storage.local.get("pageVisits"))["pageVisits"];
-    let pageId: string;
-    for (const visit of pageVisits) {
-      const visitUrl = new URL(visit.url);
-      if (visitUrl.origin === originUrl.origin && visit.tabId === tabId) {
-        pageId = visit.pageId;
+    // Record this pixel event sighting so it can be matched up with navigation events later.
+    const foundPixel = { url: url.toString(), originUrl: originUrl.toString(), tabId: tabId.toString(), hasFacebookLoginCookies, formData };
+    const foundPixels = (await browser.storage.local.get("foundPixels"))["foundPixels"];
```

Comment:
Hey!  Can we name `foundPixels` something a bit more distinct from `foundPixel`?  That's asking for typo errors.  `storedPixels` perhaps?